Requirements in Building-Transmission.md are out of date

According to [Building-Transmission.md](https://github.com/transmission/transmission/blob/main/docs/Building-Transmission.md) the requirements are:
* macOS 11.0 or newer
 * Xcode 12.5.1 or newer

My system satisfies these requirements:

    $ sw_vers -productVersion
    11.7.10

    $  xcodebuild -version
    Xcode 13.2.1
    Build version 13C100

The build process fails with the same error in multiple files:

    /tmp/src/transmission/libtransmission/api-compat.cc:11:10: fatal error: 'ranges' file not found
    #include <ranges>
            ^~~~~~~~

This appears to be an intentional design choice as referenced by this ticket: https://github.com/transmission/transmission/issues/5854#issuecomment-1660658169 . If so, the requirements are now out of date and need updating.

--- a/README.md
+++ b/README.md
@@ -32,8 +32,8 @@ For a more detailed description, and dependencies, visit [How to Build Transmiss
 ### Building a Transmission release from the command line
 
 ```bash
-$ tar xf transmission-4.0.6.tar.xz
-$ cd transmission-4.0.6
+$ tar xf transmission-4.1.0.tar.xz
+$ cd transmission-4.1.0
 # Use -DCMAKE_BUILD_TYPE=RelWithDebInfo to build optimized binary with debug information. (preferred)
 # Use -DCMAKE_BUILD_TYPE=Release to build full optimized binary.
 $ cmake -B build -DCMAKE_BUILD_TYPE=RelWithDebInfo

--- a/docs/Building-Transmission.md
+++ b/docs/Building-Transmission.md
@@ -2,9 +2,12 @@
 The source code for both official and nightly releases can be found on our [download page](https://transmissionbt.com/download/).
 
 ## On macOS ##
-Software prerequisites:
+While the prerequisites for running the app are:
  * macOS 11.0 or newer
- * Xcode 12.5.1 or newer
+
+The actual software prerequisites for building the app are based on GitHub Actions [macos-14](https://github.com/actions/runner-images/blob/main/images/macos/macos-14-arm64-Readme.md):
+ * macOS 14.8.3 or newer
+ * Xcode 15.0.1 or newer
 
 Building the project on Mac requires the source to be retrieved from GitHub. Pre-packaged source code will not compile.
 ```bash

